web_src/css/repo.css

```
@@ -942,7 +942,7 @@
   margin-bottom: -0.25rem;
 }
 
-@media (max-width: 767.98px) {
+@media (max-width: 964px) {
```

That seems like a pretty arbitrarily chosen constant.
Judging by the remaining code, it should probably be
```suggestion
@media (max-width: 991.98px) {
```
instead

---

Thank you for the review. I have replaced it with 991.98px.